db/models/user.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define("User", {
+    email: DataTypes.STRING,
+    sha1: DataTypes.STRING,
```

Comment:
I added this `sha1` field for users to hold a pre-computed SHA1 hash for faster lookups.